chromium_src/chrome/browser/ui/autofill/payments/save_card_bubble_controller_impl.cc

```diff
@@ -0,0 +1,12 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "../../../../../../../chrome/browser/ui/autofill/payments/save_card_bubble_controller_impl.h"
+
+#define BRAVE_SHOULD_SHOW_SIGNIN_PROMO_SHORT_CIRCUIT_TO_FALSE \
```

Comment:
I'd also like to do a followup to look at using sync settings to disable this instead of the patch cc @darkdh

Reply:
This will be done in https://github.com/brave/brave-core/pull/2010

Reply:
addressed in 1418b7447e0de61989f92a296383973534581c3b